Tools/WinMLRunner/src/BindingUtilities.h

```diff
@@ -168,17 +187,16 @@ namespace BindingUtilities
                                                     static_cast<int32_t>(width), static_cast<int32_t>(height));
     }
 
-    SoftwareBitmap LoadImageFile(const TensorFeatureDescriptor& imageDescriptor, InputDataType inputDataType,
-                                 const hstring& filePath, const CommandLineArgs& args, uint32_t iterationNum)
+    SoftwareBitmap LoadImageFile(const ILearningModelFeatureDescriptor& modelFeatureDescriptor,
```

Comment:
Same here. It looks like both of these functions are called from CreateBindableImage, and I think it's better to do type check there before we call these functions